test/index.js

```diff
@@ -91,20 +91,25 @@ describe('lune', function () {
   })
 
   describe('#phase_range', function () {
-    const NAMES = ['new', 'q1', 'full', 'q3']
+    const PHASES = [
+      lune.PHASE_NEW,
+      lune.PHASE_FIRST,
+      lune.PHASE_FULL,
+      lune.PHASE_LAST
+    ]
 
     /* http://aa.usno.navy.mil/data/docs/JulianDate.php */
     it('should return all moon phases within a time range', function () {
       let error = 0
 
-      for (let name of NAMES) {
+      for (let phase of PHASES) {
         const actual = lune.phase_range(
           new Date('1989-01-01T00:00Z'),
           new Date('1989-10-01T00:00Z'),
-          name
+          phase
         )
         const expected = observations.filter(function (x) {
-          return x[2] === name
+          return x[2] === phase
```

Comment:
Good call. I have a bad habit of using dumb variable names for iterators :)

Fixed.